docs/quickstart.md

```diff
@@ -50,32 +48,32 @@ PostgreSQL database at localhost:5432.
 ```shell
 #!/usr/bin/env sh
 
-VERSION=4
+VERSION=6
 
-#Get the Gateway Jar
-curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
-
-#Start the persistence DB
-if ! $(command -v psql >/dev/null); then
-    if ! $(command -v brew >/dev/null); then
-        echo 'Please install psql and rerun';
-        exit 1
-    else
-        brew install libpq
-    fi
+#Check and get the Gateway Jar
+if [[ -f "gateway-ha.jar" ]]; then
+    echo "Trino Gateway Jar already exists so skip fetching."
+else
+    echo "Trino Gateway Jar does not exist, try to fetch from maven repository."
```

Comment:
Failed to find gateway-ha.jar in current directory. Fetching version $VERSION from Maven Central repository.